pkg/kvstore/store/store_test.go

```diff
@@ -195,6 +195,31 @@ func (s *StoreSuite) TestStorePeriodicSync(c *C) {
 	c.Assert(expect(func() bool { return localKey2.deleted >= 1 }), IsNil)
 }
 
+func (s *StoreSuite) TestStoreLocalKeyProtection(c *C) {
```

Comment:
The unit test will depend on changes that are not present in 1.3. I think we can consider to skip adding the unit test

Reply:
I've removed this test since there isn't an "Observer" type.